### PR TITLE
Changes for 202012.01 release

### PR DESCRIPTION
**Pull request by aggarw13**
Changes for 202012.01 release include addition of Jobs, OTA and Defender libraries

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,18 +3,21 @@ FreeRTOS offers feature stability with long term support (LTS) releases. FreeRTO
...
+| FreeRTOS Kernel             | 10.4.3  | 03/31/2023 | https://github.com/FreeRTOS/FreeRTOS-Kernel/tree/V10.4.3                    |
+| FreeRTOS-Plus-TCP           | 2.3.2   | 03/31/2023 | https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/tree/V2.3.2                   |
+| coreMQTT                    | 1.1.0   | 03/31/2023 | https://github.com/FreeRTOS/coreMQTT/tree/v1.1.0                            |
+| coreHTTP                    | 2.0.0   | 03/31/2023 | https://github.com/FreeRTOS/coreHTTP/tree/v2.0.0                            |
+| corePKCS11                  | 3.0.0   | 03/31/2023 | https://github.com/FreeRTOS/corePKCS11/tree/v3.0.0                          |
+| coreJSON                    | 3.0.0   | 03/31/2023 | https://github.com/FreeRTOS/coreJSON/tree/v3.0.0                            |
+| backoffAlgorithm            | 1.0.0   | 03/31/2023 | https://github.com/FreeRTOS/backoffAlgorithm/tree/v1.0.0                    |
+| AWS IoT Device Shadow       | 1.0.2   | 03/31/2023 | https://github.com/aws/Device-Shadow-for-AWS-IoT-embedded-sdk/tree/v1.0.2   |
```
Comment (abhidixi11) [inline README.md:17]: Shadow has to be 1.0.2 same as before.

File: .gitmodules [hunk 2]
```diff
@@ -25,7 +25,16 @@
 	path = FreeRTOS/backoffAlgorithm
 	branch = main
 	url = https://github.com/FreeRTOS/backoffAlgorithm.git
-
 [submodule "FreeRTOS/FreeRTOS-Kernel"]
 	path = FreeRTOS/FreeRTOS-Kernel
 	url = https://github.com/FreeRTOS/FreeRTOS-Kernel.git
+[submodule "aws/jobs-for-aws-iot-embedded-sdk"]
+	path = aws/jobs-for-aws-iot-embedded-sdk
+	url = git@github.com:aws/Jobs-for-AWS-IoT-embedded-sdk.git
```
Comment (leegeth) [inline .gitmodules:33]: https links?

File: README.md [hunk 3]
```diff
@@ -3,18 +3,21 @@ FreeRTOS offers feature stability with long term support (LTS) releases. FreeRTO
...
+| AWS IoT Device Defender     | 1.1.0   | 03/31/2023 | https://github.com/aws/Device-Defender-for-AWS-IoT-embedded-sdk/tree/v1.1.0 |
+| AWS IoT Jobs                | 1.1.0   | 03/31/2023 | https://github.com/aws/Jobs-for-AWS-IoT-embedded-sdk/tree/v1.1.0            |
+| AWS IoT Over-the-air Update | 3.0.0   | 03/31/2023 | https://github.com/aws/ota-for-aws-iot-embedded-sdk/tree/v3.0.0             |
 
 ## FreeRTOS LTS Versioning and Patches
```
Comment (yngki) [inline README.md:23]: We need to add a line here for

202012.01: Added AWS IoT Device Defender, IoT Jobs, IoT OTA to FreeRTOS 202012-LTS.

File: .gitmodules [hunk 4]
```diff
@@ -25,7 +25,16 @@
...
+	branch = main
```
Comment (leegeth) [inline .gitmodules:34]: is branch required?

File: .gitmodules [hunk 5]
```diff
@@ -25,7 +25,16 @@
...
+[submodule "aws/device-defender-for-aws-iot-embedded-sdk"]
+	path = aws/device-defender-for-aws-iot-embedded-sdk
+	url = git@github.com:aws/Device-Defender-for-AWS-IoT-embedded-sdk.git
```
Comment (leegeth) [inline .gitmodules:37]: https?

File: .gitmodules [hunk 6]
```diff
@@ -25,7 +25,16 @@
...
+[submodule "aws/ota-for-aws-iot-embedded-sdk"]
+	path = aws/ota-for-aws-iot-embedded-sdk
+	url = git@github.com:aws/ota-for-aWS-iot-embedded-sdk.git
```
Comment (leegeth) [inline .gitmodules:40]: https? is aWS in the link with the correct cases?

File: .gitmodules (hunk 4 above)
Comment (aggarw13) [reply .gitmodules:33]: Other library submodules have the branch name mentioned, so this is just being consistent with them.
In the future, if we add automation functionality of auto-updating the submodules, the tracking branch will be useful

File: .gitmodules (hunk 6 above)
Comment (aggarw13) [reply .gitmodules]: Yeah, the link will still work haha. I have fixed the issue though.

File: .gitmodules (hunk 2 above)
Comment (aggarw13) [reply .gitmodules]: Resolved

File: .gitmodules (hunk 5 above)
Comment (aggarw13) [reply .gitmodules]: Updated

**Review by leegeth — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (aggarw13) [reply README.md:23]: Per separate conversation, I have added a CHANGELOG.md file for this information

File: README.md [hunk 7]
```diff
@@ -3,22 +3,25 @@ FreeRTOS offers feature stability with long term support (LTS) releases. FreeRTO
...
 
-FreeRTOS LTS releases use a date-based versioning scheme (YYYYMM) followed by a patch sequential number (.XX). For example, FreeRTOS 202012.02 LTS means the second patch to the December-2020 FreeRTOS LTS release. Currently, there are no patches for the FreeRTOS 202012.00 LTS release. You can subscribe to [GitHub notifications](https://docs.github.com/en/free-pro-team@latest/github/managing-subscriptions-and-notifications-on-github/about-notifications) to receive information on patches or other updates to this repository.   
+FreeRTOS LTS releases use a date-based versioning scheme (YYYYMM) followed by a patch sequential number (.XX). For example, FreeRTOS 202012.02 LTS means the second patch to the December-2020 FreeRTOS LTS release. You can subscribe to [GitHub notifications](https://docs.github.com/en/free-pro-team@latest/github/managing-subscriptions-and-notifications-on-github/about-notifications) to receive information on patches or other updates to this repository.
```
Comment (yngki) [inline README.md:24]: One minor suggestion:

You can review the changelog and subscribe

**Review by leegeth — APPROVED** [review]

File: README.md (hunk 7 above)
Comment (aggarw13) [reply README.md]: Updated

**Review by tanmoysen — APPROVED** [review]

**Review by leegeth — APPROVED** [review]

**Review by leegeth — APPROVED** [review]

**Pull request merged**